css/src/components/tabs.scss

```
@@ -0,0 +1,111 @@
+$tabs-padding: $font-size-9 $font-size-5 !default;
+$tabs-padding-lg: $font-size-7 $font-size-4 !default;
+
+$tab-font-color: $primary !default;
+$tab-border: $border-width solid transparent !default;
+$tabs-border: $border-width solid $border !default;
+$tabs-hover-background-color: $primary-background !default;
+$tabs-hover-border: $border-width solid $border !default;
+$tabs-hover-border-color: $primary-hover !default;
+$tabs-active-hover-background-color: $primary-hover !default;
+$tabs-active-hover-border-color: $primary-hover !default;
+$tabs-active-background-color: $primary-active;
+$tabs-active-border-color: $primary-active !default;
+$tabs-active-color: $text-invert !default;
+
+.tabs {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $font-size-7;
+	grid-template-columns: max-content auto max-content;
+	white-space: nowrap;
+}
+
+.tab-previous,
+.tab-next {
+	display: flex;
+	align-items: center;
+	padding: $tabs-padding;
+	border: $tab-border;
+	border-radius: $border-radius-rounded;
+	background-color: $tabs-active-background-color;
+	color: $tabs-active-color;
+	font-weight: $weight-semibold;
+	box-shadow: $box-shadow-medium;
+
+	&:hover {
+		border-color: $tabs-active-hover-border-color;
+		background-color: $tabs-active-hover-background-color;
+	}
+}
+
+.tab-list {
+	grid-column: 2;
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+	border-radius: $border-radius-rounded;
+	box-shadow: $box-shadow-medium;
+	overflow-x: hidden;
+
+	@include forced-colors {
+		border: $tabs-border;
+		forced-color-adjust: preserve-parent-color;
+	}
+}
+
+.tab-control {
+	display: flex;
+	position: relative;
+	align-items: center;
+	justify-content: center;
+	padding: $tabs-padding;
+	border: $tab-border;
+	border-radius: $border-radius-rounded;
+	background-color: transparent;
+	color: $tab-font-color;
+	font-weight: $weight-semibold;
+	cursor: pointer;
+	appearance: none;
+	vertical-align: top;
+
+	&:hover {
+		border-color: $tabs-hover-background-color;
+		background-color: $tabs-hover-background-color;
+		z-index: $zindex-hover;
+
+		@include forced-colors {
+			background-color: SelectedItem;
+			color: SelectedItemText;
```

These should probably be `Highlight` and `HighlightText`, which are the modern standards' replacements for these selected item system colors:

```suggestion
			background-color: Highlight;
			color: HighlightText;
```

---

updated